dask_jobqueue/jobqueue.yaml

```diff
@@ -79,3 +79,25 @@ jobqueue:
     env-extra: []
     resource-spec: null
     job-extra: []
+
+  lsf:
+    name: dask-worker
+
+    # Dask worker options
+    threads: 2
+    processes: 4
+    memory: 8GB
+    interface: null
+    death-timeout: 60
+    local-directory: null
+
+    # LSF resource manager options
+    queue: null
+    project: null
+    walltime: '00:30:00'
+    extra: ""
+    env-extra: []
+    job-cpu: null
+    job-mem: null
```

Comment:
I wonder, can we use the `memory` and `threads * processes` entries to remove the need for these entries?  This is probably also a question for other dask-jobqueue maintainers, as I know that these appear in other configurations.